### PR TITLE
feat: Application funding government projects/information about institution

### PR DESCRIPTION
**Pull request by Thorrik58**
# Information about institution screen added

https://app.clickup.com/t/hb4cfb

## What

Added institution information screen with a repeater

## Why

Step needed for applicant to define institution data.

## Screenshots / Gifs
image

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: libs/application/templates/funding-government-projects/src/forms/FundingGovernmentProjectsForm.ts [hunk 1]
```diff
@@ -39,10 +40,42 @@ export const FundingGovernmentProjectsForm: Form = buildForm({
...
+            buildCustomField(
+              {
+                id: 'organizationOrInstitutionName.description',
+                component: 'FieldDescription',
+                title: '',
+              },
+              {
+                description:
+                  informationAboutInstitution.general
+                    .infoInstitutionTextFieldDescription.defaultMessage,
```
Comment (atligudl) [inline libs/application/templates/funding-government-projects/src/forms/FundingGovernmentProjectsForm.ts:60]: This won't translate

File: libs/application/templates/funding-government-projects/src/lib/dataSchema.ts [hunk 2]
```diff
@@ -1,7 +1,24 @@
...
+  organizationOrInstitutionName: z
+    .string()
+    .nonempty(error.general.nameError.defaultMessage),
+  contacts: z.array(
+    z.object({
+      name: z.string().nonempty(error.general.nameError.defaultMessage),
+      phoneNumber: z.string().refine((v) => v && v.length >= 7, {
+        params: error.general.invalidPhoneNumber,
+      }),
+      email: z.string().refine((v) => isValidEmail(v), {
```
Comment (atligudl) [inline libs/application/templates/funding-government-projects/src/lib/dataSchema.ts:17]: This can be replaced by setting the text field variant as email

**Review by atligudl — APPROVED** [review]

File: libs/application/templates/funding-government-projects/src/lib/dataSchema.ts (hunk 2 above)
Comment (Thorrik58) [reply libs/application/templates/funding-government-projects/src/lib/dataSchema.ts:17]: Validation seems doesnt seem to work without some validation in data schema.

File: libs/application/templates/funding-government-projects/src/forms/FundingGovernmentProjectsForm.ts (hunk 1 above)
Comment (Thorrik58) [reply libs/application/templates/funding-government-projects/src/forms/FundingGovernmentProjectsForm.ts]: Fixed by using FieldTitle component

**Pull request merged**